docs/sql-ref-syntax-qry-sampling.md

```diff
@@ -19,4 +19,72 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `TABLESAMPLE` statement is used to sample the table. It supports the following sampling methods:
+- `TABLESAMPLE`(x `ROWS`): Sample the table down to the given number of rows.
+- `TABLESAMPLE`(x `PERCENT`): Sample the table down to the given percentage. Note that percentages are defined as a number between 0 and 100.
+- `TABLESAMPLE`(`BUCKET` x `OUT OF` y): Sample the table down to a `x` out of `y` fraction.
+
+Note: `TABLESAMPLE` returns the approximate number of rows or fraction requested.
```

Comment:
We need a link to this section from the `SELECT` document?
https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-qry-select.md#L78-L85

In the PostgreSQL doc, the `SELECT` document includes the `TABLESAMPLE` clause: https://www.postgresql.org/docs/current/sql-select.html

Reply:
Thanks for your comments. 
In the related statement, I will cross link ```select``` and ```tablesample```. I am a little hesitate to add tablesample to the ```from item```.  Seems to me, Tablesample is not parallel to Table relation or InlineTable, the structure is more like this:
```
SELECT ... FROM 
    Table relation
       Tablesample
    ......
    Inlined table
       Tablesample
```
Here is how snowflake document this https://docs.snowflake.com/en/sql-reference/constructs/from.html

Reply:
ok, thanks for the check. Looks ok as it is.